src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -169,17 +169,31 @@ export class InputDatePicker
     }
   }
 
-  /** Expand or collapse when calendar does not have input */
+  /**
+   * Expand or collapse when calendar does not have input
+   *
+   * @deprecated use open instead
+   */
   @Prop({ mutable: true, reflect: true }) active = false;
 
   @Watch("active")
-  activeHandler(): void {
-    if (!this.disabled || !this.readOnly) {
-      this.reposition();
+  activeHandler(value: boolean): void {
+    this.open = value;
+  }
+
+  /** Expand or collapse when calendar does not have input */
+  @Prop({ mutable: true, reflect: true }) open = false;
+
+  @Watch("open")
+  openHandler(value: boolean): void {
+    this.active = value;
```

Comment:
Wouldn't this cause an infinite loop? one prop changing the value of others and vice versa.